inst/include/common/model.hpp

```diff
@@ -74,18 +74,18 @@ class Model {  // may need singleton
     // vector< vector<Type> > creates a nested vector structure where
     // each vector can be a different dimension. Does not work with ADREPORT
     // fleets
-    vector<vector<Type> > exp_index(n_fleets);
-    vector<vector<Type> > exp_catch(n_fleets);
-    vector<vector<Type> > cnaa(n_fleets);
-    vector<vector<Type> > cwaa(n_fleets);
-    vector<vector<Type> > F_mort(n_fleets);
+      tmbutils::vector<tmbutils::vector<Type> > exp_index(n_fleets);
+      tmbutils::vector<tmbutils::vector<Type> > exp_catch(n_fleets);
+      tmbutils::vector<tmbutils::vector<Type> > cnaa(n_fleets);
+      tmbutils::vector<tmbutils::vector<Type> > cwaa(n_fleets);
+      tmbutils::vector<tmbutils::vector<Type> > F_mort(n_fleets);
```

Comment:
this can be simplified to vector<vector<Type>>

Reply:
working on this